docker/run_worker.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -e
+exec ./manage.py celery worker -A laalaa --concurrency=$WORKER_APP_CONCURRENCY --loglevel=INFO
```

Comment:
Could the log level also come from the environment variable?

Reply:
I’m wondering if this argument means the code for celery configuration is incomplete... (django does take log level into account)

Reply:
I'm not sure what you mean here. I can't see references to loglevel in settings/base.py file

Reply:
In that case I’m wondering why do we have a log level environment variable...

Reply:
@exonian has said that it could be used by Django